src/modules/blockchainApplication/explore/components/BlockchainApplicationDetails/BlockchainApplicationDetails.js

```diff
@@ -85,8 +96,8 @@ const BlockchainApplicationDetails = ({ history, location, application }) => {
             <a
```

Comment:
https://docs.google.com/document/d/1fId3uEv1xLfXxBI3sipIGPWl4YiN9caheuQIRZ68VVk/edit#
```suggestion
          blockchain.explorers.map(({url}) =>  <a
              className={`${styles.appLink}`}
              target="_blank"
              href={url}
            >
              <Icon name="chainLinkIcon" className={styles.hwWalletIcon} />
              {url}
            </a>)
```